instrumentation/grpc-1.6/library/src/test/java/io/opentelemetry/instrumentation/grpc/v1_6/MetadataSetterTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.grpc.v1_6;
+
+import static org.junit.Assert.assertTrue;
+
+import io.grpc.Metadata;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.grpc.v1_6.propagator.GrpcPropagator;
+import org.junit.jupiter.api.Test;
+
+class MetadataSetterTest {
+
+  @Test
+  void checkThatIterableSizeEqualsOne() {
+    GrpcPropagator tested = new GrpcPropagator();
```

Comment:
I would define a more meaningful name here, if it's hard to do here, is the `propagator` a better choice than now?

Reply:
I've already change the name